keyboards/handwired/pteron/keymaps/default/keymap.c

```diff
@@ -0,0 +1,105 @@
+#include "pteron.h"
```

Comment:
Could you replace this include with `#include QMK_KEYBOARD_H` instead? 

And the action_layer include is not needed, and should be removed.